src/main/java/ua/bank/bankservice/service/AccountService.java

```diff
@@ -0,0 +1,14 @@
+package ua.bank.bankservice.service;
+
+import java.util.List;
+import ua.bank.bankservice.model.Account;
+
+public interface AccountService {
+    Account create(Account account);
```

Comment:
how about having a single `save()` method here?
and when we will need to block account - set `active` to false in controller and call `save()` as well?